.github/workflows/preview-env-deploy.yml

```diff
@@ -105,12 +91,12 @@ jobs:
           env_url: https://${{ steps.secrets.outputs.PREVIEW_ENV_BUCKET_NAME }}/pr-${{ github.event.number }}/index.html
 
       - name: Update comment with deployment status
-        uses: peter-evans/create-or-update-comment@v4
+        uses: mshick/add-pr-comment@v2
         env:
           BUCKET_NAME: ${{ steps.secrets.outputs.PREVIEW_ENV_BUCKET_NAME }}
         with:
-          comment-id: ${{ steps.find-build-comment.outputs.comment-id }}
-          issue-number: ${{ github.event.number }}
-          body: |
-            The preview environment relating to the commit ${{ github.sha }} has successfully been deployed. You can access it at [https://${{ env.BUCKET_NAME }}/pr-${{ github.event.number }}/index.html](https://${{ env.BUCKET_NAME }}/pr-${{ github.event.number }}/index.html) <!-- preview-env -->
-          edit-mode: replace
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          message-id: preview-env-status
```

Comment:
`message-id` is also not necessary, if you don't plan on having more than one comment you want to update throughout the process. There's a default value like `add-pr-comment`.